### PR TITLE
feat(AUTH-CPC-211) verify if user is registered

### PR DESCRIPTION
**Pull request by ArianneBonneau**
- added dto for login user
- created mapper dto -> model
- get user by email (if exist)
- verify that password matches email 
- tested everything (100% for test coverage)

### CODE REVIEW

File: auth-service/src/main/java/com/petclinic/auth/User/UserController.java [hunk 1]
```diff
@@ -62,4 +62,11 @@ public void passwordReset(@PathVariable long id,  @RequestBody String pwd) throw
         userServ.passwordReset(id,pwd);
         log.info("Password for User with id {} with new password {}", id, pwd);
     }
+
+    @GetMapping
+     public Boolean verifyUser (@RequestBody UserIDLessUsernameLessDTO loginUser) throws NotFoundException{
```
Comment (Trilikin21) [inline auth-service/src/main/java/com/petclinic/auth/User/UserController.java:67]: if you want to verify if the user is registered, why request the body when you can request the ID and verify it inside the database?

Comment (Trilikin21) [reply auth-service/src/main/java/com/petclinic/auth/User/UserController.java:67]: Unless you verify by other values such as the username.

File: auth-service/src/main/java/com/petclinic/auth/User/UserRepo.java [hunk 2]
```diff
@@ -21,6 +21,8 @@
 public interface UserRepo extends JpaRepository<User, Long> {
 
     User findByEmail(String email);
+//    UserIDLessUsernameLessDTO findUserByEmail(String email);
```
Comment (Trilikin21) [inline auth-service/src/main/java/com/petclinic/auth/User/UserRepo.java:24]: remove commented code

**Review by Trilikin21 — APPROVED** [review]
If I understand correctly, you are trying to verify if the user is registered with the email. If that is the case. Ignore the comments about the changes requested in the service. Remove the commented code inside the Repo. Approve.

**Review by Funnyadd — APPROVED** [review]
lgtm

**Review by Trilikin21 — APPROVED** [review]
LGTM

**Review by Funnyadd — APPROVED** [review]
Lgtm

**Pull request merged**